test/mock/server.go

```diff
@@ -0,0 +1,253 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/gorilla/mux"
+)
+
+var name string
+var port string
+
+var data map[string]string
+
+func init() {
+	// name of the mock server
+	name = os.Getenv("MOCK_NAME")
+	if len(name) == 0 {
+		panic("This server must have a name, found in MOCK_NAME env")
+	}
+
+	// port of the mock server
+	port = os.Getenv("MOCK_PORT")
```

Comment:
there is an util method for returning env vars / assigning default values to them